### DISCUSSION TITLE
Pull request for externals possible?

### DISCUSSION THREAD

**Discussion by vdawg-git** [Thinkmill/keystatic#838] [Q&A]
Hey, I am searching for no db, markdown git CMS system for an Open-Source project. 
Contributors should be able to easily create PRs via the CMS which then can get reviewed.

Is this possbile with Keystatic? It looks so good and I want to try it :D

I skimmed through the docs and searched the issues/discussions, but wasnt able to find anything about it, so I am asking here :)

**Comment by silveltman** [comment]
same question here

**Comment by SultanZain** [comment]
BUMP: +1, There is a CMS as you want named DecapCMS that allow open-authoring for public repos, but the UI is crap, I hope KeyStatic will be having this feature in future.

**Comment by Greenheart** [comment]
Related issue #1433
